specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimtenant.json

```diff
@@ -130,15 +183,15 @@
           }
         }
       },
-      "patch": {
```

Comment:
for a put, the request body should be the entire resource body contract, which I believe is AccessInformationContract (same as the response)

Reply:
Not in this case. Put works as patch and allows to modify one property only, others stay unchanged. 
Put does not create any entities, they are pre-created on service activation.

Reply:
as per https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc009-use-put-for-update-of-entire-resource-ie-replace-use-patch-only-for-partial-update-of-resource and https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy006-follow-the-right-semantics-for-put-and-patch-operations - you should not be using PUTs for partial updates - even if the resource is auto-created, PUT will be used only to "replace" not "update" and  should require the full resource body

Reply:
fixed